packages/widgets/src/blocks/legacy-widget/edit/widget-type-selector.js

```diff
@@ -27,8 +27,7 @@ export default function WidgetTypeSelector( { selectedId, onSelect } ) {
 
 	return (
 		<SelectControl
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
Activate a legacy theme like Twenty Twelve. Go to Appearance ▸ Widgets, and insert a Legacy Widget block.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/45a7bb1e-c877-42b9-9e8a-c2ab9d276980" alt="Legacy Widget block placeholder, before" width="684">|<img src="https://github.com/user-attachments/assets/75e82cd8-e260-43e0-ae46-8bf77e238e97" alt="Legacy Widget block placeholder, after" width="684">|